CONTRIBUTING.md

```diff
@@ -62,6 +62,12 @@ see [ABI Policy Guide](sycl/doc/ABIPolicyGuide.md) for more information.
 - For any DPC++-related commit, the `[SYCL]` tag should be present in the
   commit message title. To a reasonable extent, additional tags can be used
   to signify the component changed, e.g.: `[PI]`, `[CUDA]`, `[Doc]`.
+- For product changes which require modification of E2E tests
```

Comment:
It probably makes sense to replace E2E here with more generic: "tests, outside of this repository"

Reply:
done